corehq/apps/registration/utils.py

```diff
@@ -113,7 +113,7 @@ def request_new_domain(request, form, is_new_user=True):
         new_domain.save()  # we need to get the name from the _id
 
     with transaction.atomic():
-        if is_new_user:
+        if is_new_user and not settings.ENABLE_NEW_TRIAL_EXPERIENCE:
             # Only new-user domains are eligible for Advanced trial
```

Comment:
the 30 day advanced trial will no longer be available

Reply:
Similar to other question, are you planning to remove the trial code as part of this rollout?

Reply:
the 30 day trial gets removed later in this PR. The trial itself is still applicable. Now we set up custom trials specifically for projects based on their needs.